src/content/overview/developer-portal/_index.md

```diff
@@ -1,12 +1,33 @@
 ---
 title: Developer portal
-description: The central Giant Swarm self-service portal for your platform engineers and developers.
+description: Our Backstage-based developer portal is your engineer's front end to the platform. We provide our self-service user interface as plugins for Backstage, so engineers using your platform find all the information they need in the right place.
 weight: 30
 menu:
   principal:
     parent: overview
     identifier: overview-developer-portal
-last_review_date: 2024-03-18
+last_review_date: 2024-07-01
 owner:
-  - https://github.com/orgs/giantswarm/teams/sig-product
+  - https://github.com/orgs/giantswarm/teams/team-honeybadger
 ---
+
+Our developer portal absed on [Backstage](https://www.cncf.io/projects/backstage/) is your engineer's front end to the platform. We provide our self-service user interface as plugins for Backstage, so that engineers using your platform find all the information they need in the right place.
```

Comment:
Reworded a lot. I don't see how "Backstage plugin" relates to finding stuff in the right place (and that last part of the sentence can be left off without losing information, i.e. "fill words" in my review-nit lingo, because a frontend always tries to put stuff in some right place).

```suggestion
The Giant Swarm platform's developer portal, based on [Backstage](https://www.cncf.io/projects/backstage/), is a frontend for customer's engineers to work with the platform. The self-service user interface is provided as plugins for Backstage.
```

Reply:
I clarified the sentence. The idea is that by integrating our UI in the portal, devs find the information we provide in the place they visit often anyway.